Cargo.toml

```diff
@@ -445,3 +456,7 @@ required-features = ["transforms-wasm", "transforms-lua"]
 
 [patch.'https://github.com/tower-rs/tower']
 tower-layer = "0.3"
+
+[patch.crates-io]
+# TODO: switch to crates.io after fix for https://github.com/Arnavion/k8s-openapi/issues/70 is released
+k8s-openapi = { git = "https://github.com/timberio/k8s-openapi.git", branch = "fix-70" }
```

Comment:
It looks like we can point to the original repo instead of a fork https://github.com/Arnavion/k8s-openapi/commit/f506c01134b5280faaa1d003c7445178df7fa36e

Reply:
Correct. I'm using fork as a fail-safe in case upstream changes. I can update the fork to be in-line with the upstream since the fix is now in their mainline.
Ultimately, we'll just need to switch to crates.io when available, until then I wanted to keep the code pointing to our fork - it's more or less under our control, and we can preserve it for historical reasons even is upstream goes away.

Reply:
I mean that we could point to that specific SHA in the upstream repo, not just the master branch in general. That makes it a bit clearer what we rely on and we can go ahead and get rid of the fork. I'm not sure that we need to guard against a commit on the master branch of the upstream repo disappearing.

Reply:
Ah, yeah, good idea! I'll update the ref to the sha.
If you think it's fine - I'll also remove our work and switch to upstream.

Reply:
Sounds good!